WindowsStubs/WindowsStubs/TaskUtil.cpp

```diff
@@ -0,0 +1,85 @@
+#include <windows.h>
+#include <cstdio>
+#include <tchar.h>
+#include <iostream>
+#include <string>
+#include <cctype>
+#include <algorithm>
+#include <memory>
+
+
+struct ProcessInfoDeleter {
+    void operator()(PROCESS_INFORMATION* ppi) {
+        if (ppi)
+        {
+            if(ppi->hProcess)
+            CloseHandle(ppi->hProcess);
+            if(ppi->hThread)
+            CloseHandle(ppi->hThread);
+            delete ppi;
+        }
+    }
+};
+
+bool TaskExecuteAndWait(std::string execpath, std::string *pOutput) {
+    STARTUPINFO si;
+    SECURITY_ATTRIBUTES securityAttr;
+    HANDLE inputPipe, outputPipe;
+    bool retval = true;
+    std::unique_ptr< PROCESS_INFORMATION, ProcessInfoDeleter > upi(new PROCESS_INFORMATION);
+    PROCESS_INFORMATION *ppi = upi.get();//only for clarity purposes, ownership is not relinquished
+    ZeroMemory(&si, sizeof(si));
+    si.cb = sizeof(si);
+    ZeroMemory(ppi, sizeof(PROCESS_INFORMATION));
+    ZeroMemory(&securityAttr, sizeof(securityAttr));
+    securityAttr.nLength = sizeof(securityAttr);
+    securityAttr.bInheritHandle = TRUE;
+    int len;
+    int slength = (int)execpath.length() + 1;
+    len = MultiByteToWideChar(CP_ACP, 0, execpath.c_str(), slength, 0, 0);
+    auto ubuf = std::unique_ptr< wchar_t[]>{ new wchar_t[len] };
+    wchar_t *pbuf = ubuf.get(); //only for clarity
+    MultiByteToWideChar(CP_ACP, 0, execpath.c_str(), slength, pbuf, len);
```

Comment:
What's expected origin (and encoding) of execpath?
Is CP_ACP suitable for our case?
Whatever we decide, it may be appropriate to add some comment in header file to avoid confusion.